components/gateway/internal/gqlschema/schema.graphql

```diff
@@ -395,8 +419,13 @@ type Mutation {
     deleteAPI(id: ID!): APIDefinition
     refetchAPISpec(apiID: ID!): APISpec
 
-    setAPICredential(apiID: ID!, in: CredentialInput!): Credential!
-    deleteAPICredential(apiID: ID!): Credential
+    """
```

Comment:
![Screen Shot 2019-05-30 at 10 17 58](https://user-images.githubusercontent.com/7155799/58619018-3a033380-82c4-11e9-8966-b10c0bfd43ff.png)

This is how it looks in Playground. We should use sentences ended with a dot.